tests/unit/src/test/scala/tests/cli/BaseCliSuite.scala

```diff
@@ -46,4 +49,132 @@ abstract class BaseCliSuite extends FunSuite with DiffAssertions {
       onStdout(stdout)
     }
   }
+
+  def assertOneSameOrPrintExpected(obtained: String, expected: List[String], title: String = "")(
```

Comment:
Why do we need to copy-paste so much code?

Reply:
As per the comments (see lines 72 and 73) the `scala.meta.testkit.DiffAssertions` methods that I needed cannot be accessed because they are private. These belong to the [testkit](https://mvnrepository.com/artifact/org.scalameta/testkit_2.12/4.3.0) package. If these were accessible much of the copy & paste is not required.  Alternatively the `assertOneSameOrPrintExpected`could be placed in the `testkit`package. 

Seeing as it is not part of MDoc I opted for this *solution*. Any way `testkit` can be changed?

Reply:
What functionality is missing? We should avoid copy pasting this amount of code

Reply:
Not required anymore so it was removed. See my explanation below. Initially I used this to test the pre and post processing methods. You correctly pointed out they were  not necessary. So I removed them but did not correct the test.